CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -48,7 +48,7 @@ public IList<HarvestDeveloper> RetrieveDevelopers()
             var apiResponse = response.Result;
             var users = apiResponse["users"];
            
-            return users.Where(user => (bool)user["is_active"] != false)
+            return users.Where (user => (bool)user["is_active"] != false && CheckUserRole(user))
```

Comment:
```suggestion
            return users.Where (user => (bool)user["is_active"] != false && IsDeveloper(user))
```
If we decide to keep this method, I think we should rename it to something more readable.

Reply:
```suggestion
            return users.Where (user => !((bool)user["is_active"]) && CheckUserRole(user))
```
😉

Reply:
applied the fix to https://github.com/chubberlisk/crypto_tech/pull/30/commits/55f45f507bf7296b62455a671f2ab24d79c75f36 for now